contracts/DssDeploy.sol

```diff
@@ -257,6 +257,8 @@ contract DssDeploy {
         vat.rely(address(pot));
         flap.rely(address(vow));
         flop.rely(address(vow));
+
+		pot.rely(msg.sender); // KEYDONIX added
```

Comment:
Different indentation styles here.  If this is code we control, I recommend following the `.editorconfig` (tabs).  If this is third party code, then use whatever they use.  It is a bit awkward wen it is 99% third party code with minor modifications, but we should be consistent within the file at least.

Reply:
This code came as spaces from upstream (like basically everything nowadays (yes I know you disagree)), your .editorconfig forced me into tabs without me realizing it!

Reply:
If you're going to commit a .editorconfig, it should probably be on you to update the files at that time

Reply:
I'm 👍 on just converting all of the files we are pulling from elsewhere to tabs.  I'll do it after this is all merged.

Reply:
Well we should make sure it's kinda easy to re-do, since we'll prolly copy in dss many more times

Reply:
The other option is to tell `.editorconfig` to use spaces for those files.  Disadvantage is we have to list them all out, or put them in a folder together.  Advantage is that when we merge upstream changes in it will "do the right thing".  I'm open to any of these options (exception being mixed indentation in the file).